rustls-platform-verifier/Cargo.toml

```diff
@@ -51,6 +51,10 @@ android_logger = { version = "0.13", optional = true } # Only used during testin
 once_cell = "1.9"
 webpki-roots = "0.25"
 
+# BSD targets require webpki-roots for the real-world verification tests.
+[target.'cfg(all(unix, not(target_os = "linux"), not(target_os = "android"), not(target_os = "macos"), not(target_os = "ios")))'.dev-dependencies]
```

Comment:
I think that `cfg(target_os = "freebsd")` would be clearer here. If we add testing support for the other 2 BSD platforms Rust supports, we could add them specifically here.

Reply:
Good idea, fixed :-)